tasks/main.yml

```diff
@@ -367,22 +416,30 @@
   package:
     name: tuned-profiles-mssql
     state: present
+  when:
+    - mssql_manage_tuned
```

Comment:
Change to a one-line when expression: `  when: mssql_manage_tuned`

Reply:
and add `| bool` - `when: mssql_manage_tuned | bool`
https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#ansible-guidelines
"Use the `| bool` filter when using bare variables (expressions consisting of just one variable reference without any operator) in `when`." and `changed_when` and `failed_when` and `assert: that`